app/src/main/java/org/wikipedia/edit/WikiTextKeyboardView.kt

```diff
@@ -142,7 +142,7 @@ class WikiTextKeyboardView : FrameLayout {
             if (title.wikiSite.languageCode != baseLangCode) {
                 text += ":" + title.wikiSite.languageCode + ":"
             }
-            text += StringUtil.fromHtml(title.displayText).toString() + "]]"
+            text += title.prefixedText + "|" + StringUtil.fromHtml(title.displayText).toString() + "]]"
```

Comment:
This will work nicely, but can we do it only if `prefixedText` and `displayText` are different?
(not including spaces/underlines, e.g. "Barack Obama" and "Barack_Obama" should be the same.)

Reply:
It makes sense to me. Done!